core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/EntityStatisticsCacheMonitor.scala

```diff
@@ -52,26 +57,25 @@ trait EntityStatisticsCacheMonitor extends LazyLogging {
   val standardPollInterval: FiniteDuration
   val workspaceCooldown: FiniteDuration
 
-  def sweep() = {
+  def sweep(): Future[EntityStatisticsCacheMessage] = {
     trace("EntityStatisticsCacheMonitor.sweep") { rootSpan =>
       dataSource.inTransaction { dataAccess =>
         // calculate now - workspaceCooldown as the upper bound for workspace last_modified
         val maxModifiedTime = nowMinus(workspaceCooldown)
-        //Note: Ignored workspaces have a cacheLastUpdated timestamp of 1000ms after epoch
-        val minCacheTime = new Timestamp(1000)
 
-        dataAccess.workspaceQuery.findMostOutdatedEntityCacheAfter(minCacheTime, maxModifiedTime).flatMap {
-          case Some((workspaceId, lastModified)) =>
+        dataAccess.entityCacheQuery.findMostOutdatedEntityCacheAfter(MIN_CACHE_TIME, maxModifiedTime).flatMap {
+          case Some((workspaceId, lastModified, cacheLastUpdated)) =>
             rootSpan.putAttribute("workspaceId", OpenCensusAttributeValue.stringAttributeValue(workspaceId.toString))
             traceDBIOWithParent("updateStatisticsCache", rootSpan) { _ =>
               DBIO.from(updateStatisticsCache(workspaceId, lastModified).map { _ =>
-                logger.info(s"Updated entity cache for workspace $workspaceId. Cache was ${System.currentTimeMillis() - lastModified.getTime}ms out of date.")
+                val outDated = lastModified.getTime - cacheLastUpdated.getOrElse(MIN_CACHE_TIME).getTime
+                logger.info(s"Updated entity cache for workspace $workspaceId. Cache was ${outDated}ms out of date.")
```

Comment:
not super necessary for this PR, but tweaking this log entry seemed like a good idea